pages/example-6-xss/solution-2.tsx

```diff
@@ -0,0 +1,41 @@
+import { BlogPost, getSpecialBlogPosts } from '../../database/blogPosts';
+import { CommonContent } from './common';
+import DOMPurify from "dompurify"
```

Comment:
- we use single quotes (make sure your Prettier + ESLint extensions are installed / configured in your VS Code - you can follow [the UpLeveled System Setup guide](https://github.com/upleveled/system-setup/blob/main/readme.md) if you are unsure which extensions to install)
- we need the TypeScript types for this package

I'll take this over